lib/Wikia/tests/Service/User/Auth/HeliosCookieHelperTest.php

```diff
@@ -59,4 +61,128 @@ public function testClearAuthenticationCookie() {
 		$this->cookieHelper->clearAuthenticationCookie( $this->response );
 	}
 
+	public function testGetAccessTokenFromCookie() {
+		$token = 'qi8H8R7OM4xMUNMPuRAZxlY';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "qi8H8R7OM4xMUNMPuRAZxlY" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)